djangocms_fil_bootstrap/components/workflows.py

```diff
@@ -30,8 +30,12 @@ def workflows(self, roles):
 
     def workflow(self, name, data, roles):
         steps = data.pop("steps", [])
-        workflow = Workflow.objects.create(**data)
-        for step in steps:
-            step["role"] = roles[step["role"]]
-            workflow.steps.create(**step)
-        self.data[name] = workflow
+        workflow_data = {
+            "is_default": data["is_default"],
+        }
+        workflow, created = Workflow.objects.get_or_create(name=data["name"], defaults=workflow_data)
+        if created:
```

Comment:
There is no test that would check this `if` does not run when `created == False`.

Reply:
This assert fails if `created == false`

```
 def test_parse_workflow(self):
        ...
        self.assertEqual(component.data["wf1"].steps.count(), 1)
```